exampleCourse/questions/demo/randomFakeData/server.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+import random
+
+from faker import Faker
+
+
+def formatDate(date):
+    return date.strftime("%B %d, %Y")
+
+
+def appendEmployee(data, fake, alias, birthdays):
```

Comment:
Nit: snake_case for Python conventions.

```suggestion
def append_employee(data, fake, alias, birthdays):
```